lib/jsanalyze.js

```diff
@@ -14,6 +14,7 @@
 'use strict';
 
 const appc = require('node-appc');
+const path = require('path');
```

Comment:
<!--
  0 failure: 
  1 warning:  lib/jsanalyze.js ...
  
  
  DangerID: danger-id-Danger;
  File: lib/jsanalyze.js;
  Line: 17;
-->

- :warning: lib/jsanalyze.js line 17 – 'path' is assigned a value but never used. (no-unused-vars)